sdks/java/README.md

```
@@ -0,0 +1,71 @@
+
+try (Queue queue = Taskito.builder().backend("sqlite").url("taskito.db").open()) {
+    String id = queue.enqueue(sendEmail, Map.of("to", "a@b.c"));
+    Job job = queue.getJob(id).orElseThrow();   // job.status == JobStatus.PENDING
+    QueueStats stats = queue.stats();
+    queue.cancel(id);
+}
+```
+
+## Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)